lightning/src/routing/network_graph.rs

```diff
@@ -41,6 +41,11 @@ pub struct NetworkGraph {
 	nodes: BTreeMap<PublicKey, NodeInfo>,
 }
 
+/// A simple newtype for RwLockReadGuard<'a, NetworkGraph>.
+/// This exists only to make accessing a RwLock<NetworkGraph> possible from
+/// the C bindings, as it can be done directly in Rust code.
+pub struct LockedNetworkGraph<'a>(pub RwLockReadGuard<'a, NetworkGraph>);
```

Comment:
Hm, should we start using this in RL, just to get some test coverage on it?

Reply:
Its used in the bindings `demo.cpp` :shrug: